namer/consul/src/main/scala/io/buoyant/namer/consul/DcServices.scala

```diff
@@ -60,19 +86,22 @@ private[consul] object DcServices {
                 case Some(svc) => svc
                 case None =>
                   log.debug("consul added: %s", k)
-                  SvcAddr(consulApi, name, k, domain)
+                  stats.adds.incr()
+                  SvcAddr(consulApi, name, k, domain, stats.service)
```

Comment:
Prefer to start conservative with number of metrics to start.  What we have here should be enough to do relative debugging ("huge spike in service changes", etc).  If this ends up being too opaque, we can always add more stats.  My gut says this is Good Enough.